ansible/roles/grouper/templates/iplant-grouper.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description={{grouper.service_description}}
+BindsTo=docker.service
+PartOf=docker.service
+After=docker.service
+Requisite=docker.service
+
+[Service]
+ExecStartPre=-/usr/bin/docker rm -v {{grouper.container_name}}
+ExecStart=/usr/bin/docker run --name {{grouper.container_name}} \
+    -p 8080:8080 \
+    -v /etc/grouper:/etc/grouper \
+    -v /etc/tomcat/server.xml:/opt/tomcat/conf/server.xml \
+    -v /etc/tomcat/tomcat-users.xml:/opt/tomcat/conf/tomcat-users.xml \
+    -v /etc/tomcat/setenv.sh:/opt/tomcat/bin/setenv.sh \
+    -v /var/log/grouper:/var/log/grouper \
+    -v /etc/localtime:/etc/localtime -v /etc/timezone:/etc/timezone \
+    --log-driver={{grouper.log_driver}} \
+    {{remote_registry}}/{{grouper.image_name}}:{{docker.tag}}
+ExecStop=/usr/bin/docker stop {{grouper.container_name}}
+Restart=on-failure
+StartLimitInterval=60s
+StartLimitBurst=3
+
+SyslogIdentifier={{grouper.container_name}}
+
+[Install]
+WantedBy={{systemd.target}}
+
```

Comment:
Files have been removed. Rebasing onto dev helped identify what was needed to be removed.